www/contribute/team.html

```diff
@@ -0,0 +1,38 @@
+---
+layout: team
+title: Team
+---
+
+<div class="row">
+    <div class="col-sm-12 text-center">
+        <h1>Team</h1>
+        <p>These are some of people behind Apache Cordova</p>
+    </div>
+</div>
+
+{% for member in site.members %}
+<div class="row">
+    <div class="col-sm-8 col-xs-8">
+        <span>
+            <h2>{{ member.name }}</h2>
+            <p>{{ member.subtitle }}</p>
+        </span>
+    </div>
+</div>
+<div class="row">
+    <div class="col-sm-8">
+        <p>
+            {{ member.content }}
+        </p>
+    </div>
+    <div class="col-sm-3 col-sm-offset-1">
+        <img src="{{ member.imageurl }}"><img>
+        {% if member.twitter %}
+            <a href="{{ member.twitter }}">Twitter</a>
```

Comment:
Are we able to use a twitter icon here for the hyperlink instead of having a textual link or does this raise trademark/copyright concerns?

I think graphically an icon is better than a textual link.

FontAwesome 4.7 is a free set of icons related under MIT (Code)/SIL (Font files) licenses. I think it's safe to use this for as long as we update our [attributions](https://cordova.apache.org/attributions/) page. FontAwesome has a [twitter](https://fontawesome.com/v4.7.0/icon/twitter) icon, among other social media platforms.

Reply:
To be hones I don't want to figure out how to include another icon set just for this. I also thought doing this dynamically for include like 3 custom links with their own labels. If that's a good idea we can do that maybe later?

I don't like working on this because the built is just painfully slow.

Reply:
Sounds good to me